app/helpers/application_helper.rb

```diff
@@ -52,15 +52,37 @@ def base_title
     'Operation Paws for Homes'
   end
 
+  def index_page_title
```

Comment:
a better name would be `dog_index_page_title`

I like moving this to a helper though, that's what they're there for right?  Maybe do a dog specific helper instead since these seem to be mostly for dogs.  No strong feelings either way.

Reply:
Yeah, a helper would definitely be the "Rails Way". I even thought about that at the time, but helpers always feel to me like a surreptitious way of incorporating logic into views, and then hiding it, and not having a good testing strategy. My preference is always "fat models". I think Rails needs to have a View Model concept baked-in. I'll move this to a helper.

Reply:
wait... what am I saying? It is in a helper. I guess a dogs-specific helper for the #index_page_title method is what you're saying.